### ISSUE TITLE
Questions about Sync BatchNorm

### ISSUE THREAD

**Issue by ecr23xx** [StanfordVL/MinkowskiEngine#64]
Is there any improvement of changing `ME.MinkowskiBatchNorm` to `ME.MinkowskiSynchBatchNorm`? They both use `nn.BatchNorm1d`, and the real batch size is the number of points in a batch, which is very large in one scene (> 50k). So I guess the improvement of using sync BN will not be very significant? Can you provide some benchmark about those two BN?

Thanks.

**Comment by chrischoy** [comment]
Hi,

I think even if there are a large number of points in a batch, the diversity of the points would be small if the batch size is small per GPU. However, I agree that it would not make a big difference. Let me get back to you with the results.
